config/settings.yml

```diff
@@ -52,10 +52,12 @@ session_cookie:
 
 sign_in:
   jwt_encode_key: 'spec/fixtures/sign_in/privatekey.pem'
-  redirect_uri: 'vamobile://login-success'
-  web_redirect_uri: 'http://localhost:3001/auth/login/callback'
   cookies_secure: false
   info_cookie_domain: localhost
+  client_redirect_uris:
+    mobile: 'vamobile://login-success'
+    web: 'http://localhost:3001/auth/login/callback'
+    mobile_test: 'http://localhost:4001/auth/sis/login-success'
```

Comment:
I was under the impression it would need to be hardcoded to `https://va-mobile-cutter.herokuapp.com/auth/sis/login-success`.

Reply:
This was intentional for localhost, as per Patrick's comment in our Slack thread. The `herokuapp` setting will be added to the staging `settings.yml` config in the devops repo, so it will redirect to that URL on staging as you'd expect

Reply:
That makes sense. Thank you.